deps.sh

```diff
@@ -24,6 +25,11 @@ elif [ "$UNAME" = "Linux" ]; then
     elif [ "$HAS_PACMAN" -eq 1 ]; then
         pacman -Syy
         pacman -S libzip libcurl-gnutls pkg-config || exit 1
+    elif [ "$HAS_YUM" -eq 1 ]; then
+        yum makecache
+        yum install -y libcurl-devel libzip-devel pkgconfig || exit 1
+    else
+        echo "Unknown Linux distribution"
```

Comment:
Similar to what we're doing below with `Unknown platform`

```suggestion
        echo "Unknown Linux distribution"
        exit 1
```